internal/controller/ingress/tlsedge_controller.go

```diff
@@ -356,7 +355,7 @@ func (r *TLSEdgeReconciler) updateIPRestrictionModule(ctx context.Context, edge
 	return err
 }
 
-func (r *TLSEdgeReconciler) listTLSEdgesForIPPolicy(obj client.Object) []reconcile.Request {
+func (r *TLSEdgeReconciler) listTLSEdgesForIPPolicy(ctx context.Context, obj client.Object) []reconcile.Request {
```

Comment:
should we use this `ctx` in this func?

Reply:
I'll look into that! My goal was to leave the internals of the funcs as unchanged as possible, but it's probably better to use it anyway.

Reply:
used context here: https://github.com/ngrok/kubernetes-ingress-controller/pull/328/commits/de9ea688a6db70bbd404b01f5d1ae5780639d9af